lib/ansible/modules/cloud/scaleway/scaleway_compute.py

```diff
@@ -296,13 +296,14 @@ def create_server(compute_api, server):
             "tags": server["tags"],
             "commercial_type": server["commercial_type"],
             "image": server["image"],
-            "dynamic_ip_required": server["dynamic_ip_required"],
             "name": server["name"],
             "organization": server["organization"]
             }
 
-    if server["security_group"]:
-        data["security_group"] = server["security_group"]
+    # Update optional parameters
+    for optional_param in ("security_group", "dynamic_ip_required", "public_ip"):
```

Comment:
Could you put those variables in a special variable? It looks like a constant that might be used elsewhere.